agents/test/MarriageAgentE2E.java

```diff
@@ -36,7 +36,7 @@ public void testMarriageAgentInsertsTheExpectedNumberOfMarriages() {
 
                 List<Numeric> answer = tx.execute(marriagesCountQuery);
                 int numMarriages = answer.get(0).number().intValue();
-                int expectedNumMarriages = 119;
+                int expectedNumMarriages = 115;
```

Comment:
Are we going to have to do this every time new agents are added? That seems like an issue to me.

I think the solution we originally talked about was to assert that at least X marriages are inserted, or otherwise within an expected range.

An alternative would be to run the simulation with only the given agent included.

Reply:
Absolutely agree, it's an issue, but the lightest weight testing that would demonstrate not regressing on determinism. Let's chat about this IRL